test/framework/functional/output_cloudwatch.go

```diff
@@ -31,7 +30,7 @@ var (
 	route     *openshiftv1.Route
 )
 
-func (f *CollectorFunctionalFramework) AddCloudWatchOutput(b *runtime.PodBuilder, obs obs.OutputSpec) error {
+func (f *CollectorFunctionalFramework) AddCloudWatchOutput(b *runtime.PodBuilder, spec obs.OutputSpec) error {
```

Comment:
`spec` is unused and can be removed:

```suggestion
func (f *CollectorFunctionalFramework) AddCloudWatchOutput(b *runtime.PodBuilder) error {
```

Reply:
exists for consistency with the other types.  The proper design would probably be using some sort of interface and callback

Reply:
I think I would go with simplicity over consistency here. We're not using any interface here and I don't think it's necessary to introduce one, so just passing the needed information to the functions seems fine.